### PR TITLE
TQ42-1572: added notebook with cva_opt example using channels

### PR DESCRIPTION
**Pull request by ChristopheFoussette**

### CODE REVIEW

File: notebooks/cva_opt_with_channels_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"device_code\":\"v-YmkmEoO3Hny9AkO9DZ-eTg\",\"user_code\":\"CRWR-ZNTN\",\"verification_uri\":\"https://auth.terraquantum.io/activate\",\"expires_in\":900,\"interval\":5,\"verification_uri_complete\":\"https://auth.terraquantum.io/activate?user_code=CRWR-ZNTN\"}\n",
```
Comment (sbtq) [inline notebooks/cva_opt_with_channels_example.ipynb:62]: you want to clean the outputs (or at least these corresponding to authentication data)

File: notebooks/cva_opt_with_channels_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "5bbbd0d2-4863-459b-a25f-bf4424fc94d3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Running experiment within: Org 4ae7d0af-80f3-4272-a9b2-c3e17c3594b9, Proj 81bf18de-5a56-4e7c-8534-6ea46402f90e and Exp 1c921f32-f425-43cf-8318-4f3bf6a3a9ce`\n"
```
Comment (sbtq) [inline notebooks/cva_opt_with_channels_example.ipynb:89]: not necessarily important but disclosing your ids with this notebook is not the best practice.

File: notebooks/cva_opt_with_channels_example.ipynb [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " starting run with id 11bc7584-c3f0-4c34-9820-206643ccb903\n",
+      "run is in state RUNNING, continuing\n"
+     ]
+    }
+   ],
+   "source": [
+    "#run = await run_in_loop(tq42client, experiment_id, cva_params)\n",
```
Comment (sbtq) [inline notebooks/cva_opt_with_channels_example.ipynb:298]: this can be removed?

File: notebooks/cva_opt_with_channels_example.ipynb [hunk 4]
```diff
@@ -0,0 +1,435 @@
...
+    "        )\n",
+    "    # define a function to be called after the optimization is finished\n",
+    "    def success():\n",
+    "        pass\n",
+    "\n",
+    "    # let the channel wait for connections\n",
+    "    is_finished = False\n",
+    "    max_retries = 10\n",
+    "    retries = 0\n",
+    "    while (not is_finished) and (retries < max_retries):\n",
```
Comment (sbtq) [inline notebooks/cva_opt_with_channels_example.ipynb:265]: next week with the new feature I will be working on this should get way better. But for now I would be fine with doing this workaround.

**Review by sbtq — CHANGES_REQUESTED** [review]

File: notebooks/cva_opt_with_channels_example.ipynb (hunk 1 above)
Comment (ChristopheFoussette) [reply notebooks/cva_opt_with_channels_example.ipynb:62]: Yes, I will clear all outputs.

**Review by sbtq — APPROVED** [review]

**Pull request merged**